gamla/functional.py

```diff
@@ -775,6 +775,14 @@ def drop(seq):
     return drop
 
 
+def flip(func: Callable):
+    @currying.curry
+    def flip(a, b):
+        return toolz.flip(func, a, b)
```

Comment:
why use toolz :-) ? just call func(b,a) here (this is what toolz does)

Reply:
😆 good point.. thx